### PR TITLE
Update bo.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added some channel  fixed some stream

#EXTINF:-1 tvg-logo="https://i.imgur.com/dWtxNYu.png" group-title="BOLIVIA",Red Uno
http://136.243.3.70:1935/RedUno/RedUno/chunklist_w372299009.m3u8
#EXTINF:-1 group-title="Bolivia" tvg-logo="https://i.imgur.com/7lkIFVg.jpg" group-title="BOLIVIA",Unitel
http://136.243.3.70:1935/Unitel/Unitel/chunklist_DVR.m3u8
#EXTINF:-1 group-title="Bolivia" tvg-logo="https://i.imgur.com/APwaLsO.jpg" group-title="BOLIVIA",Católica TV
http://136.243.3.70:1935/CatolicaTv/CatolicaTv/playlist.m3u8

those dont seem to work

### CODE REVIEW

File: channels/bo.m3u [hunk 1]
```diff
@@ -1,27 +1,53 @@
...
+#EXTINF:-1 tvg-id="ATB.bo" tvg-name="ATB" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/DCAtokY.png" group-title="",ATB (360p)
 http://186.121.206.197/live/daniel/index.m3u8
-#EXTINF:-1 tvg-id="ATBCochabamba.bo" tvg-name="ATB Cochabamba" tvg-country="BO" tvg-language="Spanish" tvg-logo="" group-title="",ATB Cochabamba
+#EXTINF:-1 tvg-id="ATBCochabamba.bo" tvg-name="ATB Cochabamba" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/oszItYJ.jpg" group-title="",ATB Cochabamba (360p)
 https://mediacp.hostradios.com.ar:19360/atbcochabamba/atbcochabamba.m3u8
-#EXTINF:-1 tvg-id="ATBLaPaz.bo" tvg-name="ATB La Paz" tvg-country="BO" tvg-language="Spanish" tvg-logo="" group-title="",ATB La Paz
+#EXTINF:-1 tvg-id="ATBLaPaz.bo" tvg-name="ATB La Paz" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/oszItYJ.jpg" group-title="",ATB La Paz (340p)
 https://clientes.videostreaming.ar:19360/atbbolivia/atbbolivia.m3u8
+#EXTINF:-1 tvg-id="ATB.bo" tvg-name="ATB" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/oszItYJ.jpg" group-title="",ATB Santa Cruz
+https://mediacp.hostradios.com.ar:19360/atbcom/atbcom.m3u8
```
Comment (freearhey) [inline channels/bo.m3u:9]: broken link
Captura de pantalla 2021-04-09 a las 16 24 59

File: channels/bo.m3u [hunk 2]
```diff
@@ -1,27 +1,53 @@
...
 #EXTINF:-1 tvg-id="BoliviaTV.bo" tvg-name="Bolivia TV" tvg-country="BO" tvg-language="Spanish" tvg-logo="http://www.boliviatv.bo/principal/assets/images/the-voyager/btv1.png" group-title="",Bolivia TV (720p)
 http://boliviatv1.srfms.com:5735/live/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="boliviatv.bo" tvg-name="Bolivia TV 1" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="",Bolivia TV 7.1 (720p)
+https://5f2c1b0d880e5.streamlock.net/boliviatv/videoboliviatv/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9NC85LzIwMjEgNDozMDozOCBBTSZoYXNoX3ZhbHVlPXRQanZQSlJJZi9RQW82c2M0SllwTkE9PSZ2YWxpZG1pbnV0ZXM9NTAmaWQ9
```
Comment (freearhey) [inline channels/bo.m3u:13]: broken link
Captura de pantalla 2021-04-09 a las 16 25 34

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

**Review by Sphinxroot — COMMENTED** [review]
corrected   1 have a smal change in token  sadly

they other one will check later https://www.atb.com.bo/atbenvivo/  stream down on site

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Thanks!

**Comment by kone78** [comment]
I add 2 Spain Channels

#EXTINF:0
#EXTINF:-1 tvg-logo="https://static.wikia.nocookie.net/logopedia/images/7/7d/Antena_3_2017.svg/revision/latest?cb=20200307025520&path-prefix=es" group-title="SPAIN",ANTENA 3 HD  
http://hls.webredirect.org/gR5t/mp_8495.php/https://directo.atresmedia.com/1617998463_c11c3fbb4b958ab43eefba403bc7ead6d571936e/antena3/master.m3u8
#EXTINF:0
#EXTINF:-1 tvg-logo="https://images.app.goo.gl/UXxZGEW8yq31Mqx1A
"group-title="SPAIN",ENERGY 
https://live.mitele.es/hls/live/2005337/energy/playlist.m3u8

**Comment by freearhey** [comment]
@kone78 ANTENA 3 HD added via https://github.com/iptv-org/iptv/commit/d9764d00b60f3f4e275121b3f176dd7f499b7935. The link to ENERGY is not working at the moment.
Captura de pantalla 2021-04-09 a las 21 21 55

**Pull request merged**